packages/@aws-cdk/custom-resources/README.md

```diff
@@ -369,6 +369,15 @@ const awsCustom2 = new AwsCustomResource(this, 'API2', {
 })
 ```
 
+### Error Handling
+
+Every error produced by the API call is treated as is and will cause a "FAILED" response to be submitted to CloudFormation. You can ignore some errors by specifying the `ignoreErrorCodesMatching` property, which accepts a regular expression that is tested against the `code` property of the response. If matched, a "SUCCESS" response is submitted. Note that in such a case, the call response data, and the `Data` key submitted to CloudFormation, will both be an empty JSON object. This presents us with some limitations:
```

Comment:
Too many comma's:

```suggestion
Every error produced by the API call is treated as is and will cause a "FAILED" response to be submitted to CloudFormation. You can ignore some errors by specifying the `ignoreErrorCodesMatching` property, which accepts a regular expression that is tested against the `code` property of the response. If matched, a "SUCCESS" response is submitted. Note that in such a case, the call response data and the `Data` key submitted to CloudFormation would both be an empty JSON object. Since a successful resource provisioning might or might not produce outputs, this presents us with some limitations:
```

Reply:
👍